package.json

```diff
@@ -6,14 +6,21 @@
   "scripts": {
```

Comment:
I've added them based on domain (travis/ci, coverage, etc), but alphabetically is ok too.